2019-02-18-root-systems-meeting.md

```diff
@@ -0,0 +1,147 @@
+# 2019-02-18 Toronto Workers Co-op: Root Systems Meeting
+
+<sup>[from template][template]</sup>
+
+[:date: Calendar][cal] | [:cat: GitHub][gh]
+
+Time: Mon, Feb 18 @ 11pm ET  
+Location: https://appear.in/offline  
+Attending: patcon, mikey@enspiral/root, benhylau, dan@bamboo/root
+ 
+[TOC]
+
+## Agenda
+
+- check-ins
+- questions
+
+## Summary
+
+- RootSys (consulting + product) shutting down.
+- Plan for exit strategies. Don't assume owners will stay forever.
+- Lack of accountability systems. People weren't always working on things that brought in $$$.
+- "Membership seemed to be whoever was in the room at the time of decision." Applied to founding too. Poor idea in retrospect.
+- Struggled to have non-Dan (CEO-ish) people behave like owners.
+- Experimented with: (details below)
+    - Models for collectivizing income; member vs non-member.
+    - Basic income model.
+    - [Financial Agreement](https://www.rootsystems.nz/agreements/financial_agreement.html) (full)
+        - BEST :: basic income > commission > pay-for-time :: WORST
+    - Advice process
+- Bamboo Consulting (new, consulting only)
+    - Shared current role breakdown (details below)
+        - Success = People putting up hands to take roles. -- Dan
+- Thoughts on ideal composition of junior/int/senior devs during founding. (details below)
+    - Involve only people able to self-manage time is critical (considered "seniors")
+- Education for "how to be a co-op owner" would have been great idea! -- Dan
+- Great to be transparent externally, but focus on **accessible info** internally. -- Dan
+- Good step one: build informal network of ally orgs. Directory. -- Dan
+- "Member hiatus" policy hightly recommended so that only active members have votes. (See: Enspiral)
```

Comment:
```suggestion
- "Member hiatus" policy highly recommended so that only active members have votes. (See: Enspiral)
```

Reply:
Looks good, but for some reason I can't merge that. There's some cautious permission stuff on the "suggest changes" feature, and I think only the PR starter can merge them.

Reply:
heh assigned myself the issue and now i can :)